lib/linguist/languages.yml

```diff
@@ -3047,13 +3047,21 @@ PLSQL:
   tm_scope: none
   color: "#dad8d8"
   extensions:
-  - ".pls"
```

Comment:
Is `.pls` really no longer considered the primary extension?  

The primary extension should be the first in the list and the only one that doesn't need to be in alphabetical order.